recipes/mongo-c-driver/all/conandata.yml

```
@@ -1,4 +1,7 @@
 sources:
+  "1.29.1":
+    url: "https://github.com/mongodb/mongo-c-driver/releases/download/1.29.1/mongo-c-driver-1.29.1.tar.gz"
+    sha256: "e3dbe86e1f40e100344c48a4c0e39d2da360d2b12a54f16031fa8f90ff16fe5e"
```

```suggestion
    sha256: "64382a775353e67c510450c83d72b72e6747ba18256dd1d313deeef7db073b4f"
```

(was duplicated from 1.29.0)